docs/sources/next/shared/experimental-timers-module.md

```diff
@@ -4,8 +4,8 @@ title: Experimental timers module admonition
 
 {{% admonition type="caution" %}}
 
-Starting on k6 `v0.50`, the experimental module `k6/experimental/timers` has been graduated, and its functionality is now available in the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed in `v0.52.0`.
+The experimental module `k6/experimental/timers` has graduated, and its functionality is now globally available and through the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed.
```

Comment:
```suggestion
The experimental module `k6/experimental/timers` has graduated, and its functionality is now available globally through the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed.
```

Reply:
The functionality is available in both ways: globally and through the `k6/timers` module.

This suggestion seems to indicate that there is only one option/

Reply:
I don't think this change is necessarily indicating that there's only one option, but just steering people to use it globally. Alternatively, maybe this helps make it clearer like you had it before:

```suggestion
The experimental module `k6/experimental/timers` has graduated, and its functionality is now available globally and through the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed.
```